stingray/events.py

```diff
@@ -511,3 +515,46 @@ def apply_deadtime(self, deadtime, inplace=False, **kwargs):
             new_ev = [new_ev, retall]
 
         return new_ev
+
+    def change_mjdref(self, new_mjdref):
+        """Change the MJD reference time (MJDREF) of the light curve.
+
+        Times will be now referred to this new MJDREF
+
+        Parameters
+        ----------
+        new_mjdref : float
+            New MJDREF
+
+        Returns
+        -------
+        new_lc : lightcurve.Lightcurve object
```

Comment:
I'm assuming this is a copy-paste error and should be an `Events` object, not a `Lightcurve` object?

Reply:
Yep. Ooops

Reply:
Done